packages/ng/loading/loading.component.ts

```diff
@@ -0,0 +1,45 @@
+import { booleanAttribute, Component, effect, HostBinding, inject, input, ViewEncapsulation } from '@angular/core';
+import { LuClass } from '@lucca-front/ng/core';
+
+@Component({
+	selector: 'lu-loading',
+	standalone: true,
+	providers: [LuClass],
+	styleUrls: ['./loading.component.scss'],
+	template: `<ng-content />`,
+	encapsulation: ViewEncapsulation.None,
+	host: {
+		class: 'loading',
+	},
+})
+export class LoadingComponent {
+	#luClass = inject(LuClass);
+
+	size = input<'L' | null>(null);
+	invert = input(false, { transform: booleanAttribute });
+	block = input(false, { transform: booleanAttribute });
+	template = input<'popin' | 'drawer' | 'fullpage' | null>(null);
+
+	@HostBinding('class.mod-L')
+	get sizeInput(): boolean {
+		return this.size() === 'L';
+	}
+
+	@HostBinding('class.mod-invert')
+	get invertInput(): boolean {
+		return this.invert();
+	}
+
+	@HostBinding('class.mod-block')
+	get blockInput(): boolean {
+		return this.block();
+	}
```

Comment:
We could put all this inside the `setState` call in the effect below to have a cleaner result

Reply:
Or we could use `host: { '[class.mod-block]': 'block()' }` in the `@Component` decorator?